dashboards/resource_lakeview_dashboard.go

```diff
@@ -0,0 +1,233 @@
+package dashboards
+
+import (
+	"bufio"
+	"context"
+	"crypto/md5"
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"time"
+
+	"github.com/databricks/databricks-sdk-go/service/dashboards"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// Reads the file content from a given path
+func readFileContent(source string) ([]byte, error) {
+	log.Printf("[INFO] Reading %s", source)
+	f, err := os.Open(source)
+	if err != nil {
+		return nil, err
+	}
+	defer f.Close()
+	reader := bufio.NewReader(f)
+	return io.ReadAll(reader)
+}
+
+// Calculates MD5 hash of the given content
+func calculateMd5Hash(content []byte) string {
+	return fmt.Sprintf("%x", md5.Sum(content))
+}
```

Comment:
Same for this, I think we did this for other resources as well, can we share those?